src/main/java/com/conveyal/r5/streets/StreetRouter.java

```diff
@@ -261,9 +261,7 @@ public void setOrigin(TIntObjectMap<State> bikeStations, int switchTime, int swi
         queue.clear();
 
         bikeStations.forEachEntry((vertexIdx, bikeStationState) -> {
-            // backEdge needs to be unique for each start state or they will wind up dominating each other.
-            // subtract 1 from -vertexIdx because -0 == 0
-            State state = new State(vertexIdx, -vertexIdx - 1, streetMode);
+            State state = new State(vertexIdx, - 1, streetMode);
```

Comment:
Probably. I just change the values to be the same as before the change that broke bike sharing but since this change also doesn't fix it it's probably something else.